packages/migrations/CHANGELOG.json

```
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "5.2.0",
```

```suggestion
        "version": "6.0.0",
```

I'd advocate for a major bump here as interacting with this `Forwarder` is a breaking change.